variables.tf

```diff
@@ -50,6 +50,10 @@ variable "public_subnet_map_public_ip_on_launch" {
   default = "false"
 }
 
+variable "create_s3_vpc_endpoint" {
+  default = "true"
```

Comment:
Please add a description. We use `terraform-docs` to generate the docs in the README file.

Reply:
Ah, makes sense. Added the description.